themes/tracer/layouts/partials/otel.html

```diff
@@ -1,7 +1,11 @@
 <header class="otelHeader">
   <div class="otelHeader__container">
     <div class="otelHeader__content">
-      <p>OpenTracing and OpenCensus have merged to form <a href="https://opentelemetry.io">OpenTelemetry</a>!</p>
+      <p>The OpenTracing project is DEPRECATED.
```

Comment:
nit: can we use `<b>` or `<strong>` or similar rather than all-caps for "deprecated"? This reads like shouting for me...